components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -784,10 +785,10 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
                     if (ready_test != null) {
                         setTimeout(() => {
                             // We rotate on image addresses of v0.4 and v0.5+ t to support both v0.4 and v0.5+
-                            if (ready_test.src.includes('hub/logo')) {
+                            if (ready_test.src.includes('favicon')) {
```

Comment:
do we still need the if else test? Looks like with the line 774 update, favicon is always going to be in the src?

Reply:
We might have to iterate for v0.5 compatibility, v0.6 and v0.5 have different image location